### PR TITLE
Rules limitations

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: firestore.rules [hunk 1]
```diff
@@ -119,37 +119,42 @@ service cloud.firestore {
...
         )
         && isRequestFieldValueEqualToOrNotSetted(['id'], invitationId);
-    	allow update: if isOnlyChangingInvitationStatus(invitationId);
+    	allow update: if isOnlyChangingInvitationStatus();
       allow delete: if isInvitationSender();
     }
 
     /// FUNCTIONS ///
 
     function isInvitationSender() {
```
Comment (GrandSchtroumpf) [inline firestore.rules:135]: Could you add a comment to make it easier to make the difference beteen sender & creator ?

File: firestore.rules [hunk 2]
```diff
@@ -303,20 +301,15 @@ service cloud.firestore {
...
       allow create: if userHasValidOrg() 
         && isRequestFieldValueEqualToOrNotSetted(['id'], eventId)
         && ( isRequestFieldValueEqualTo(['ownerId'], userId()) || isRequestFieldValueEqualTo(['ownerId'], userOrgId()) );
       allow update: if userHasValidOrg() 
-        && isNotUpdatingId('id', eventId)
+        && isNotUpdatingId('id', existingData().id)
         && ( existingData().ownerId == userId() || existingData().ownerId == userOrgId() );
       allow delete: if userHasValidOrg()
-        && ( existingData().ownerId == userId() || ( existingData().ownerId == userOrgId() && isOrgAdmin(userId(), userOrgId()) ) );
+        && ( existingData().ownerId == userId() || ( existingData().ownerId == userOrgId() && isOrgAdmin(userOrgId()) ) );
```
Comment (GrandSchtroumpf) [inline firestore.rules:312]: Maybe create a function for that. It's difficult to understand

File: firestore.rules [hunk 3]
```diff
@@ -445,3 +441,6 @@ service cloud.firestore {
 
   }
 }
+
+//tester récuperer la liste des mes invités
```
Comment (GrandSchtroumpf) [inline firestore.rules:445]: Maybe remove ^^

Comment (phpgeekfr) [reply firestore.rules:445]: sure ? ^^

**Pull request merged**